openedx/tests/completion_integration/test_models.py

```diff
@@ -62,7 +62,7 @@ def setUp(self):
         self.set_up_completion()
 
     def test_changed_value(self):
-        with self.assertNumQueries(SELECT + UPDATE + 2 * SAVEPOINT + 2 * OTHER):
+        with self.assertNumQueries(2 * SELECT + UPDATE + 2 * SAVEPOINT + 2 * OTHER):
```

Comment:
Changes made in the `edx/completion` package https://github.com/openedx/completion/compare/v4.3.0...v4.4.0#diff-e2b2f2ca03394fe7e5ad502bff61da649c068416b46359f98b37729c25a71a23R44.